docs/design/lexical_conventions/symbolic_tokens.md

```diff
@@ -0,0 +1,185 @@
+# Symbolic Tokens
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Overview](#overview)
+-   [Details](#details)
+    -   [Symbolic token list](#symbolic-token-list)
+    -   [Whitespace](#whitespace)
+-   [Alternatives considered](#alternatives-considered)
+-   [References](#references)
+
+<!-- tocstop -->
+
+## Overview
+
+Symbolic tokens are a set of tokens used to represent operators. Operators are
+one use of symbolic tokens, but they are also used in patterns `:`, declarations
+(`->` to indicate return type, `,` to separate parameters), statements (`;`,
+`=`, and so on), and other places (`,` to separate function call arguments).
+
+Some languages have a fixed set of symbolic tokens, For example:
+[C++ operators](https://eel.is/c++draft/lex.operators) and
+[rust operators](https://doc.rust-lang.org/book/appendix-02-operators.html).
+While some others have extensible rules for defining operators, including the
+facility for a developer to define operators that aren't part of the
+baselanguage. For example:
+[Swift operator rules](https://docs.swift.org/swift-book/ReferenceManual/LexicalStructure.html#ID418),
+[Haskell operator rules](https://www.haskell.org/onlinereport/haskell2010/haskellch2.html#dx7-18008).
+
+Carbon has a fixed set of tokens that represent operators, defined by the
+language specification. Developers cannot define new tokens to represent new
+operators.
```

Comment:
How about changing it into something like this?
`Carbon has a fixed set of symbolic tokens, defined by the language specification and is used to represent operators. An option to define a new symbolic token doesn't exist.`

Reply:
I think we don't need "and is used to represent operators". I'd also rephrase the second sentence to be clearer that there's no way for _user code_ to define new symbolic tokens. Otherwise, it might be taken as saying that even the language designers can't add new symbolic tokens.

Reply:
`Developers cannot define new symbolic tokens in their own code`
or `An option to create user-defined tokens dosen't exists`

@geoffromer Will any of the above work?

Reply:
Will add it